spotlight/factorization/explicit.py

```diff
@@ -80,7 +82,7 @@ def __init__(self,
         self._l2 = l2
         self._use_cuda = use_cuda
         self._sparse = sparse
-        self._optimizer = None
+        self._optimizer = optimizer
```

Comment:
Ah, good call! I think there will be other issues with calling `fit` multiple times, though. For example, do you also want to have a check [here](https://github.com/EthanRosenthal/spotlight/blob/de2cd829a69f447447c1239fe1b4eb6de0ede66c/spotlight/factorization/explicit.py#L117) to see if the `_net` already exists before instantiating a new one? The other issue would be with overwriting the `_num_users` and `_num_items` attributes, but it doesn't look like those would be used for anything right now.